trestle/oscal/catalog.py

```diff
@@ -70,182 +44,36 @@ class Control(OscalBaseModel):
-        ...,
-        description='A unique ID for the location, for reference.',
-        title='Location Universally Unique Identifier',
-    )
-    title: Optional[str] = Field(
-        None,
-        description='A name given to the location, which may be used by a tool for display and navigation.',
-        title='Location Title',
-    )
-    address: Optional[Address] = None
-    email_addresses: Optional[List[EmailAddressDatatype]] = Field(
-        None, alias='email-addresses'
-    )
-    telephone_numbers: Optional[List[TelephoneNumber]] = Field(
-        None, alias='telephone-numbers'
-    )
-    urls: Optional[List[AnyUrl]] = Field(None)
-    props: Optional[List[common.Property]] = Field(None)
-    links: Optional[List[common.Link]] = Field(None)
-    remarks: Optional[str] = None
-
-
-class Party(OscalBaseModel):
-    """
-    An organization or person, which may be associated with roles or other concepts within the current or linked OSCAL document.
-    """
-
-    class Config:
-        extra = Extra.forbid
-
-    uuid: constr(regex=r'^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[45][0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$') = Field(
-        ...,
-        description='A unique identifier for the party.',
-        title='Party Universally Unique Identifier',
-    )
-    type: constr(regex=r'^\S(.*\S)?$') = Field(
-        ...,
-        description='A category describing the kind of party the object describes.',
-        title='Party Type',
-    )
-    name: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        description='The full name of the party. This is typically the legal name associated with the party.',
-        title='Party Name',
-    )
-    short_name: Optional[constr(regex=r'^\S(.*\S)?$')] = Field(
-        None,
-        alias='short-name',
-        description='A short common name, abbreviation, or acronym for the party.',
-        title='Party Short Name',
-    )
-    external_ids: Optional[List[common.ExternalId]] = Field(
-        None, alias='external-ids'
-    )
-    props: Optional[List[common.Property]] = Field(None)
-    links: Optional[List[common.Link]] = Field(None)
-    email_addresses: Optional[List[EmailAddressDatatype]] = Field(
-        None, alias='email-addresses'
-    )
-    telephone_numbers: Optional[List[TelephoneNumber]] = Field(
-        None, alias='telephone-numbers'
-    )
-    addresses: Optional[List[Address]] = Field(None)
-    location_uuids: Optional[List[constr(regex=r'^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[45][0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$')]] = Field(
-        None, alias='location-uuids'
-    )
-    member_of_organizations: Optional[List[constr(regex=r'^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[45][0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$')]] = Field(
-        None, alias='member-of-organizations'
-    )
-    remarks: Optional[str] = None
-
-
-class Metadata(OscalBaseModel):
```

Comment:
I´m assuming that metadata has been deleted automatically due to moving it to the common class right?

Reply:
For sure, Metadata is in common now. But as far as I can tell it is in common in the develop and main branches too. Not understanding why it appears here!?